### PR TITLE
fix(bare-metal): rename RV1 page and document kernel update options

### PR DESCRIPTION
**Pull request by gilbsgilbs**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- rename RV1 page
- document kernel update options

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]
Have you checked that this link isn't used anywhere else in the documentation? If we change the name of the file without updating potential docs where we refer to this page, we may have broken links.

**Review by ldecarvalho-doc — APPROVED** [review]

**Comment by gilbsgilbs** [comment]
Drafting this PR as I had a few remarks from my team.

File: bare-metal/elastic-metal/reference-content/elastic-metal-rv1-guidelines.mdx [hunk 1]
```diff
@@ -116,5 +116,23 @@ custom upgrade channel. Follow the steps below to update your kernel:
...
+    implements an unratified version of RISC-V vector instructions (RVV 0.7.1)
+    which requires specific development efforts.
+
+    For a comprehensive list of options, refer to `rv1 update-kernel --help`.
+    </Message>
 4. Wait for the update to complete. This step can take a few minutes.
 5. Reboot the server in normal mode.
+
+<Message type="note">
+If you're updating from kernel version `5.10.113-scw0` or lower, note that
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/reference-content/elastic-metal-rv1-guidelines.mdx:135]:
```suggestion
If you are updating from kernel version `5.10.113-scw0` or lower, note that
```

**Review by nerda-codes — APPROVED** [review]

File: bare-metal/elastic-metal/reference-content/elastic-metal-rv1-guidelines.mdx [hunk 2]
```diff
@@ -116,5 +116,23 @@ custom upgrade channel. Follow the steps below to update your kernel:
...
+    For a comprehensive list of options, refer to `rv1 update-kernel --help`.
+    </Message>
 4. Wait for the update to complete. This step can take a few minutes.
 5. Reboot the server in normal mode.
+
+<Message type="note">
+If you are updating from kernel version `5.10.113-scw0` or lower, note that
+newer kernels disable vector instructions by default. Refer to the above
+documentation if you still wish to keep them enabled.
+</Message>
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/reference-content/elastic-metal-rv1-guidelines.mdx:138]:
```suggestion
<Message type="note">
If you are updating from kernel version `5.10.113-scw0` or lower, note that
newer kernels disable vector instructions by default. Refer to the process above if you still wish to keep them enabled.
</Message>
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**